### ISSUE TITLE
The Teensy 3.2 is no longer available. 

### ISSUE THREAD

**Issue by Kuillemaul** [darknao/btClubSportWheel#25]
Thank you for your excellent work on this. As the 3.2 is no longer in production are there any plans to update the code to work with a 4.0?

**Comment by gernoff** [comment]
Here here.   I would like to use this firmware as well.    Could it possibly be flashed as it to the Teensy4?

**Comment by gernoff** [comment]
......... as is

**Comment by Kuillemaul** [comment]
No. I believe It makes calls that are specific to the 3.2 so it will not compile for a 4.0. I don't know enough about code or the relevant chips to modify it to work.
